airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/S3DestinationConfig.java

```diff
@@ -87,8 +88,8 @@ public static S3DestinationConfig getS3DestinationConfig(final JsonNode config)
         config.get("s3_bucket_name").asText(),
         bucketPath,
         config.get("s3_bucket_region").asText(),
-        config.get("access_key_id").asText(),
-        config.get("secret_access_key").asText(),
+        config.get("access_key_id") == null ? "" : config.get("access_key_id").asText(),
```

Comment:
let's explicitly check that these are both set or both null, and throw an error if exactly one is non-null.

Reply:
Done, if the user inputs only  id OR keys, a message with error will be presented

![image](https://user-images.githubusercontent.com/23335136/149019224-c1c2e5a5-3045-4efb-bd95-e1f3cfd1a506.png)